File: app/src/main/java/com/widget/MainActivity.java
package com.widget;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

	private Activity mActivity;

	private TextView mTextMoreLayout, mTailsTextView, mLoadView, mJRecyclerViewTest;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);
		mActivity = this;

		mTextMoreLayout = (TextView) findViewById(R.id.mTextMoreLayout);
		mTextMoreLayout.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				startActivity(new Intent(mActivity, MinePersonalSettingsActivity.class));
			}
		});

		mTailsTextView = (TextView) findViewById(R.id.mTailsTextView);
		mTailsTextView.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				startActivity(new Intent(mActivity, TailsTextLayoutTestActivity.class));
			}
		});

		mLoadView = (TextView) findViewById(R.id.mLoadView);
		mLoadView.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				startActivity(new Intent(mActivity, TestLoadView.class));
			}
		});

		mJRecyclerViewTest = (TextView) findViewById(R.id.mJRecyclerViewTest);
		mJRecyclerViewTest.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				startActivity(new Intent(mActivity, JRecyclerViewTest.class));
			}
		});
	}
}

File: app/src/main/java/com/widget/MainActivity.kt
package com.widget

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mTextMoreLayout.setOnClickListener { startActivity(Intent(this, MinePersonalSettingsActivity::class.java)) }
        mTailsTextView.setOnClickListener { startActivity(Intent(this, TailsTextLayoutTestActivity::class.java)) }
        mLoadView.setOnClickListener { startActivity(Intent(this, TestLoadView::class.java)) }
        mJRecyclerViewTest.setOnClickListener { startActivity(Intent(this, JRecyclerViewTest::class.java)) }
        mSuperText.setOnClickListener { startActivity(Intent(this, SuperTextTestActivity::class.java)) }
    }
}
